tools/bio3d/macros.xml

```diff
@@ -20,6 +19,7 @@
     </xml>
     <xml name="citations">
         <citations>
+            <citation type="doi">10.1093/bioinformatics/btz107</citation>
```

Comment:
this will now be shown on all tools, is that correct?

Reply:
Yes, We can add this right?

Reply:
Sure, if this is correct.